pyomo/core/base/param.py

```diff
@@ -878,11 +870,16 @@ def _pprint(self):
         """
         Return data that will be printed for this component.
         """
+        if self._default_val is _NotValid:
+            default = "None" # for backwards compatibility in reporting
```

Comment:
I debated this.  By returning "None", Param's pprint is consistent with previous behavior (i.e., no baselines had to be updated); however, now pprint() will not distinguish between "No default value" and a default value of None.  The downside of doing the right thing and changing pprint() to return something like "{no default}" or even "{none}" is that just about *every* pprint() baseline would change.